dev-lang/elm-compiler/elm-compiler-0.19.1.ebuild

```diff
@@ -0,0 +1,64 @@
+# Copyright 1999-2018 Gentoo Authors
+# Distributed under the terms of the GNU General Public License v2
+
+EAPI=6
```

Comment:
`EAPI=6` looks old. Can you regenerate ebuild using new hackport?

Reply:
@trofi I actually didn't use hackport for this b/c this package is not on hackage any more, at least not the latest.

Rather, I used the previous ebuild as a starting point. I've fixed the EAPI issue though.

Reply:
Looks good. I normally use `hackport make-ebuild` for non-hackage `.cabal` files.